### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by PrimeTimeTran** [huycuong0610/rubyCoderschool#1]
Cuong,
With the URL you listed in the readme.md file I cannot test your application. 
Please update and let me know when you've done it so I can test your application.

**Comment by huycuong0610** [comment]
hi I updated and push readme.md again ... If you need more update or request.Please let me know, i will take time for implement it by hard ..

Thanks sir, and have a nice day

**Comment by PrimeTimeTran** [comment]
@huycuong0610, 
Thanks, I'm going to check it out right now.

**Comment by PrimeTimeTran** [comment]
@huycuong0610,
I'm unable to log out without throwing an error on the heroku app. Could you please look into this and try to correct the bug?

Looking forward to hearing back from you.

**Comment by huycuong0610** [comment]
yes, i changed it to get method , and it working well. Thanks for your time review

**Comment by PrimeTimeTran** [comment]
@huycuong0610, when I try to destroy an article it does not do anything. When I try to delete a comment, there is an error thrown. Please work on this and get back to me

**Comment by huycuong0610** [comment]
oh , i tried to fix it . It cause by conflict javascript. Thanks sir,

**Comment by PrimeTimeTran** [comment]
@huycuong0610,
I still cannot delete articles, the app crashes when I attempt to do so.

**Comment by huycuong0610** [comment]
yes, i updated it again. @PrimeTimeTran

**Comment by PrimeTimeTran** [comment]
@huycuong0610,
[  ] Users can only edit & delete their own Articles.
A user can delete articles that other users have created.

[  ] User can only edit & delete their own comments.
Users cannot properly edit their own comments.

**Comment by huycuong0610** [comment]
@PrimeTimeTran 
Thanks for your review, I updated permission already

**Comment by PrimeTimeTran** [comment]
Ok good work and thanks~! @huycuong0610
